### ISSUE TITLE
[UI] Switch to Dark Mode by default on Meshery UI and update to User Preferences

### ISSUE THREAD

**Issue by Chadha93** [meshery/meshery#7765]
#### Current Behavior
Currently, the default mode of Meshery UI is light and has a toggle theme button in preferences but does not update the theme into preferences. 

#### Desired Behavior
Switch to Dark Mode by default on Meshery UI and update to User Preferences. 

#### Mockups

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

**Comment by devhd9** [comment]
Hey @Chadha93 , I would like to work on it. Can you assign it to me ?

**Comment by Chadha93** [comment]
> Hey @Chadha93 , I would like to work on it. Can you assign it to me ?

@devhd9 Sure, please confirm you have built Meshery locally on your machine successfully.

**Comment by anshgoyalevil** [comment]
@Chadha93 I would like to work upon this issue. Could you please assign it to me.

**Comment by anshgoyalevil** [comment]
@Chadha93 I have fixed the issue.
The default theme would now be dark, and it would get saved to user's localStorage("Theme").

> update to User Preferences.
I couldn't get that point. Could you please clarify that.

**Comment by devhd9** [comment]
> @devhd9 Sure, please confirm you have built Meshery locally on your machine successfully.

@Chadha93 , I am facing some issue with local build. I see @anshgoyalevil is working on it, I will pick another issue once my local issue is resolved.

**Comment by Chadha93** [comment]
> 

@anshgoyalevil right, we are storing in local storage, let's cover the first part only, I'll make this change directly. I'll be opening up a few issues related to dark mode bugs, let's get you assigned those.

**Comment by anshgoyalevil** [comment]
@Chadha93 Alright. Thanks! 😄
